content-src/experiments/activity-stream.yaml

```diff
@@ -18,22 +18,20 @@ contribute_url: 'https://github.com/mozilla/activity-stream'
 bug_report_url: 'https://github.com/mozilla/activity-stream/issues'
 discourse_url: 'https://discourse.mozilla-community.org/c/test-pilot/activity-stream'
 privacy_notice_url: 'https://github.com/mozilla/activity-stream/blob/master/data_dictionary.md'
-measurements: >
-  <p>In addition to the <a href="/privacy">data</a> collected by all Test Pilot
-    experiments, here are the key things you should know about what is happening
-    when you use Activity Stream:</p>
-  <ul>
-    <li>We collect basic usage data regarding how you interact with experimental
-      new tab and timeline pages, such as the types of elements that you click
-      and their position on the page.</li>
-    <li>We also collect basic information about your browser profile that affect
-      Activity Stream, such as the number of bookmarks and history items.</li>
-    <li>To provide Activity Stream, we fetch metadata about websites you have
-      visited, such as images and icons, using a service called <a
-      href="http://embed.ly/" target="_blank">Embedly</a>. We send these requests
-      to Embedly through Mozilla's servers, so they will not be associated with
-      you.</li>
-  </ul>
+measurements:
+  - >
+    We collect basic usage data regarding how you interact with experimental
+    new tab and timeline pages, such as the types of elements that you click
+    and their position on the page.
+  - >
+    We also collect basic information about your browser profile that affect
+    Activity Stream, such as the number of bookmarks and history items.
+  - >
+    To provide Activity Stream, we fetch metadata about websites you have
+    visited, such as images and icons, using a service called <a
+    href="http://embed.ly/" target="_blank">Embedly</a>. We send these requests
```

Comment:
Sure, but we're the ones who built this yaml config file thing, so we should be able to tweak it a bit to support l10n best practices.

For example, we could instead have the yaml use markdown link style (the string literal `[link text](link URL)`), then use our yaml parser to replace the link with a placeholder before sending to localizers.

So experiment authors would write something like `...a service called [Embedly](https://embedly.com)`, which is easy for them, while localizers would see two separate strings, and a [standard localization note](https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices#Add_localization_notes):
```
# LOCALIZATION NOTE(activity-stream-measurements): {{embedly}} will 
# be replaced at run-time by an active link. The string 'embedly-link' will be
# used as text for this link.
activity-stream-measurements= a service called {{embedly}}
embedly-link=Embedly
```

Finally, when templating the site, we'd just need to parse the localized strings for the double-mustachio'd placeholders, and convert into an anchor tag with the correct text infixed.

Since most of our HTML is links, I think it'd make sense to support this use case.

This is just a sketch of an idea; anything that works with l10n best practices, and keeps things manageable for experiment yaml maintainers, would do.